android/src/main/java/com/squareup/sdk/reader/react/CheckoutModule.java

```diff
@@ -184,14 +184,16 @@ static private boolean validateJSCheckoutParams(ReadableMap jsCheckoutParams, St
             return false;
         }
         if (amountMoney.hasKey("currencyCode") && amountMoney.getType("currencyCode") != ReadableType.String) {
-            paramError.append("'amount' is not an integer");
+            paramError.append("'currencyCode' is not an String");
```

Comment:
Idk if its a convention, but grammatically it should be:
> `paramError.append("'currencyCode' is not a String");`

Reply:
good catch, fixed!